lib/info-extras.js

```diff
@@ -18,10 +18,7 @@ const getMetaItem = (body, name) => util.between(body, `<meta itemprop="${name}"
 exports.getVideoDescription = html => {
   const regex = /<p.*?id="eow-description".*?>(.+?)<\/p>[\n\r\s]*?<\/div>/im;
   const description = html.match(regex);
-  if (description) {
-    return Entities.decode(decodeURIComponent(util.stripHTML(description[1])));
-  }
-  return '';
+  return description ? Entities.decode(util.stripHTML(description[1])) : '';
```

Comment:
I reverted the changes on description, and will open a separate PR with a working version.

Reply:
cool. you can create another branch on your fork. typically, people create branches for each of their features/fixes, that way you can work on many at the same time without having to wait for one of them to merge